app/api/resources/user.py

```diff
@@ -22,6 +22,8 @@
 
 DAO = UserDAO()  # User data access object
 
+# Flask mishandles boolean as string
+TRUTHY = ['true', 'True', 'yes']
```

Comment:
Personally, I would just accept `true` as value, and before checking, I would lowercase the true. I am not sure how adding this as a parameter works 🤔

Reply:
Could you link us to the relevant documentation?

Reply:
@isabelcosta I have kept 3 most used options available. In javascript `true` is used in lowercase but in python `True `with uppercase and most people get confused with it what to pass. And I wanted to keep it type-safe (without using string) so this option looks good to me. I found this somewhere on StackOverflow. 

We can also keep one small `true `and mention it in `@users_ns.doc` parameter description itself. 
Please suggest.